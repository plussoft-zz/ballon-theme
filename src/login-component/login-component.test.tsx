import React from "react";
import { shallow } from "enzyme";

import LoginComponent from "./login-component";

describe("Login Page", () => {
  let props: any;

  beforeEach(() => {
    props = {
      theme: "primary"
    };
  });

  const renderWrapper = () => shallow(<LoginComponent {...props} />);

  describe("Snapshots", () => {
    it("should match snapshots as default", () => {
      expect(renderWrapper()).toMatchSnapshot();
    });
  });
});
