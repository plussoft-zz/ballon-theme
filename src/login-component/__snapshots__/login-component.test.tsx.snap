// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Page Snapshots should match snapshots as default 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row no-gutter"
  >
    <div
      className="d-none d-md-flex col-md-4 col-lg-6 bg-image"
    />
    <div
      className="col-md-8 col-lg-6"
    >
      <div
        className="login d-flex align-items-center py-5"
      >
        <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col-md-9 col-lg-8 mx-auto"
            >
              <h3
                className="login-heading mb-4"
              >
                Welcome back!
              </h3>
              <form>
                <div
                  className="form-label-group"
                >
                  <input
                    autoFocus={true}
                    className="form-control"
                    id="inputEmail"
                    placeholder="Email address"
                    required={true}
                    type="email"
                  />
                  <label
                    htmlFor="inputEmail"
                  >
                    Email address
                  </label>
                </div>
                <div
                  className="form-label-group"
                >
                  <input
                    className="form-control"
                    id="inputPassword"
                    placeholder="Password"
                    required={true}
                    type="password"
                  />
                  <label
                    htmlFor="inputPassword"
                  >
                    Password
                  </label>
                </div>
                <div
                  className="custom-control custom-checkbox mb-3"
                >
                  <input
                    className="custom-control-input"
                    id="customCheck1"
                    type="checkbox"
                  />
                  <label
                    className="custom-control-label"
                    htmlFor="customCheck1"
                  >
                    Remember password
                  </label>
                </div>
                <button
                  className="btn btn-lg btn-primary btn-block btn-login text-uppercase font-weight-bold mb-2"
                  type="submit"
                >
                  Sign in
                </button>
                <div
                  className="text-center"
                >
                  <a
                    className="small"
                    href="#"
                  >
                    Forgot password?
                  </a>
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
